iana-mediatypes/oci-mediatype.md

```diff
@@ -0,0 +1,66 @@
+**Name:** Steve Lasker
+
+**Email:** tob@opencontainers.org
+
+**Media type name:** application
+
+**Media subtype name:** application/vnd.oci.image.manifest.v1+json
+
+**Required parameters:** n/a
+
+**Optional parameters:** n/a
+
+**Encoding considerations:** binary
+
+**Security Considerations:** This JSON format media type inherits the security considerations for JSON. See RFC 8259 section 12. (https://tools.ietf.org/html/rfc8259#section-12)
+
+**Privacy Information:** The manifest contains configuration information, with optional annotations. While users may put any content they wish into these annotations, there are no requirements for privacy information that must be protected.
+
+**Extensibility:** The json payload supports annotations as strings. No executable information or code paths are enabled.
+
+**Compression:** The manifest type is not compressed in normal workflows.
+
+**Links:** The manifest contains digests of other linked content, which are separately retrieved, with their own security considerations.
```

Comment:
> IANA: Missing from this list is integrity protection, and it appears to be important: Unless I misunderstand something, tampering with the content of a manifest could cause bad things to happen. As such, a note about the need to protect the integrity of objects of this type would seem to be needed.

Added "Content Digest spec: https://github.com/opencontainers/distribution-spec/blob/master/spec.md#content-digests" to line 29